StoreApplication/StoreApp.DataLibrary/Handlers/Repository.cs

```diff
@@ -36,18 +39,19 @@ public async Task<IEnumerable<BusinessLogic.Objects.Customer>> GetAllCustomersAs
                     zip = e.Zip
 
                 }
-            }).ToHashSet() ;
+            }).ToHashSet();
         }
-        public async Task AddCustomerAsync(BusinessLogic.Objects.Customer BLCustomer)
+
+        public async Task AddCustomerAsync(BusinessLogic.Objects.Customer blCustomer)
```

Comment:
let's add xml docs to this method while we're at it

Reply:
aH DARN